src/EPR.RegulatorService.Facade.Core/Helpers/LogHelpers.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.Extensions.Logging;
+using System.Runtime.InteropServices;
+
+namespace EPR.RegulatorService.Facade.Core.Helpers
+{
+    public static class LogHelpers
```

Comment:
Please delete this class and use _logger.Log<error\info\warn>("{Message}", message); format to resolve SQ issues
https://stackoverflow.com/questions/70955861/the-logging-message-template-should-not-vary-between-calls-ca2254-when-only-pa

https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2254

Example
_logger.LogError($"Error when creating the enrollment for id {request.UserId}");
becomes
_logger.LogError($"Error when creating the enrollment for id {UserId}", request.UserId);